### PR TITLE
NXJS-134: do not override the global Nuxeo object if already defined

### PR DESCRIPTION
**Pull request by troger**

### CODE REVIEW

File: lib/index-browserify.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+const N = require('./index');
+
+// keep back anything declared on Nuxeo object
+Object.assign(N, window.Nuxeo || {});
```
[the remark this answers is not in the document]
Comment (troger) [reply lib/index-browserify.js]: Code will be transpiled with babel, so if we need something we can add it to babel, such as https://www.npmjs.com/package/babel-plugin-transform-object-assign

[the remark this answers is not in the document]
Comment (Gabez0r) [reply lib/index-browserify.js]: Sounds good to me! 👍

[the remark this answers is not in the document]
Comment (troger) [reply lib/index-browserify.js]: I could just rely on `extend` instead of `OBject.assign` as the module is already here and loaded...

**Comment by ataillefer** [comment]
:+1:

**Comment by Gabez0r** [comment]
👍

**Pull request merged**